s3torchconnectorclient/python/src/s3torchconnectorclient/_mountpoint_s3_client.pyi

```diff
@@ -6,23 +6,23 @@ from typing import List, Optional
 # This interface is unstable!
 
 class MountpointS3Client:
-    region: str
-    endpoint: str
-    user_agent_prefix: str
     throughput_target_gbps: float
+    region: str
     part_size: int
-    profile: str
+    profile: Optional[str]
     no_sign_request: bool
+    user_agent_prefix: str
+    endpoint: str
 
     def __init__(
         self,
         region: str,
-        endpoint: str = "",
         user_agent_prefix: str = "",
         throughput_target_gbps: float = 10.0,
         part_size: int = 8 * 1024 * 1024,
-        profile: str = None,
+        profile: Optional[str] = None,
         no_sign_request: bool = False,
+        endpoint: Optional[str] = None,
```

Comment:
Why do we need to rearrange parameters order here?

Reply:
They were in the wrong order - in [MountpointS3Client](https://github.com/awslabs/s3-connector-for-pytorch/blob/main/s3torchconnectorclient/rust/src/mountpoint_s3_client.rs#L56), they're written in the new order.